src/index.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import { BrowserRouter as Router } from 'react-router-dom';
 import reportWebVitals from './reportWebVitals';
 
+// import App from './App';
 import './style.css';
+import Layout from './components/layout';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Router>
+      {/* <App /> */}
+      <Layout />
+    </Router>
```

Comment:
- [ ] Please remove the commented code and move the routing configuration to the App component instead of the layout component. The Index and app components are used in the wrong way since they are just wrapping other imported components.